### ISSUE TITLE
Invalid Class Label

### ISSUE THREAD

**Issue by rahabwairimu05** [haifengl/smile#405]
[##](url
![invalidclass2](https://user-images.githubusercontent.com/7096353/51993304-3c40b700-24b7-11e9-97a8-407350d8de80.PNG)
) Expected behaviour
Create a SVM classifier that can predict the IRIS dataset.  I am training of 150 sample dataset( Three different flowers - setosa, virginica and vesicolor), and I also tried to train it in the 100 dataset, which means I removed one flower and used 2.
The training attributes are 4, and they are supposed to classify the flower to either
0: Setosa
1:Vesicolor
2:Virginica

## Actual behaviour
The algorithm throws an exception with an error of  invalid class label:2 Here is the stack trace attached as an image.

## Code snippet
Attached as an image
E.g. The code to parse data and the call of Smile algorithm.
`	/**
	 * Fitting the Model ( Support Vector Machine)
	 */
		
		int [] predictedPrices;
		//SVM MYtrain = new SVM.Trainer(null, 0.1, 0).train(irisIndependentTrainArr, flowerSpeciesDependentTrainArr);
		
		SVM mynewSVM = new SVM.Trainer<>(null, 0, 0).train(irisIndependentTrainArr, flowerSpeciesDependentTrainArr);
	predictedPrices = mynewSVM.predict(irisIndependentTestArr);`
## Input data

E.g. sample data
